<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Baby Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="cameraLabel">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="infoLayout">
      <item>
       <widget class="QGroupBox" name="motionGroup">
        <property name="title">
         <string>motion status</string>
        </property>
        <layout class="QVBoxLayout" name="motionLayout">
         <item>
          <widget class="QLabel" name="motionStatusLabel">
           <property name="text">
            <string>unknown</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="thGroup">
        <property name="title">
         <string>temperature / humidity </string>
        </property>
        <layout class="QVBoxLayout" name="thLayout">
         <item>
          <widget class="QLabel" name="tempLabel">
           <property name="text">
            <string>temperature: -- Â°C</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="humLabel">
           <property name="text">
            <string>humidity: -- %</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="historyTabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tempHumPage">
       <attribute name="title">
        <string>history temp and humidity</string>
       </attribute>
       <layout class="QVBoxLayout" name="thChartLayout">
        <item>
         <widget class="QtCharts::QChartView" name="thChartView" native="true"/>
        </item>
        <item>
         <widget class="QChartView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="motionPage">
       <attribute name="title">
        <string>motion history</string>
       </attribute>
       <layout class="QVBoxLayout" name="motionChartLayout">
        <item>
         <widget class="QtCharts::QChartView" name="motionChartView" native="true"/>
        </item>
        <item>
         <widget class="QChartView" name="graphicsView_2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qtcharts::qchartview.h</header>
  </customwidget>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
